src/main/java/org/broadinstitute/hellbender/engine/filters/MetricsReadFilter.java

```diff
@@ -0,0 +1,38 @@
+package org.broadinstitute.hellbender.engine.filters;
+
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+public final class MetricsReadFilter implements ReadFilter {
```

Comment:
> akiezun added a note 8 days ago
 I'd prefer this placed in the ReadFilterLibrary class. You can name it something like 'MetricsFilter' or something.
-- https://github.com/broadinstitute/gatk/pull/1053#discussion-diff-43075594

Hey @akiezun - I may have misunderstood what you meant by ReadFilterLibrary. MetricsFilter seemed to belong alongside ReadFilterLibrary in the `org.broadinstitute.hellbender.engine.filters` package. Please let me know if you meant something else.